ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -338,8 +339,22 @@ function create_powervs_workspace() {
   echo "${RESOURCE_GROUP_ID}" > "${SHARED_DIR}"/RESOURCE_GROUP_ID
 
   ##Create a Workspace on a Power Edge Router enabled PowerVS zone
-  retry "ic pi workspace create ${workspace_name} --datacenter ${POWERVS_ZONE} --group ${RESOURCE_GROUP_ID} --plan public"
-  sleep 60
+  # Dev Note: uses a custom loop since we want to redirect errors
+  for i in $(seq 1 "$NO_OF_RETRY"); do
```

Comment:
Instead of I , we can have a retry_attempt as variable name . otherwise looks good to me .